frontend/public/kubevirt/components/vm.jsx

```diff
@@ -1,11 +1,21 @@
 import * as _ from 'lodash-es';
 import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
 
 import { ListHeader, ColHead, List, ListPage, ResourceRow, DetailsPage } from './factory/okdfactory';
 import { breadcrumbsForOwnerRefs, Firehose, ResourceLink, navFactory, ResourceCog, Cog } from './utils/okdutils';
-import { VirtualMachineInstanceModel, VirtualMachineModel, PodModel, NamespaceModel, TemplateModel, NetworkAttachmentDefinitionModel } from '../models';
+import {
+  VirtualMachineInstanceModel,
+  VirtualMachineModel,
+  PodModel,
+  NamespaceModel,
+  TemplateModel,
+  NetworkAttachmentDefinitionModel,
+  StorageClassModel,
+  PersistentVolumeClaimModel
+} from '../models';
 import { k8sCreate, actions } from '../module/okdk8s';
-import { connect } from 'react-redux';
+import { units } from '../../components/utils/units';
```

Comment:
Please use `frontend/public/kubevirt/components/utils/okdutils.tsx` as the reference is already indirected there.